Update hololens-calibration.md (#6225)
MicrosoftDocs/windows-itpro-docs: devices/hololens/hololens-calibration.md

- Moving objects in your visor out of the way (such as hair)
- Turning on a light in your room or moving out of direct sunlight

If you followed all guidelines and calibration is still failing, you can disable the calibration prompt in Settings. Please also let us know by filing feedback in [Feedback Hub](hololens-feedback.md).

### Calibration data and security

Calibration information is stored locally on the device and is not associated with any account information. There is no record of who has used the device without calibration. This mean new users will get prompted to calibrate visuals when they use the device for the first time, as well as users who opted out of calibration previously or if calibration was unsuccessful.

> The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be deleted automatically.

Reviewer: Please add two spaces (or a CR) to the end of the preceding paragraph. Update the new text as follows:
The device can locally store up to 50 calibration profiles. After this number is reached, the device automatically deletes the oldest unused profile.

Reviewer 2: Based on the comment above, this is the requested change:
```suggestion

The device can locally store up to 50 calibration profiles. After this number is reached, the device automatically deletes the oldest unused profile.
```

> The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be automatically deleted. 

Reviewer 2: I am fairly sure the two words at the end of this sentence are normally used in the opposite order:
```suggestion
The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be deleted automatically.
```
